sigopt/xgboost/run.py

```diff
@@ -0,0 +1,110 @@
+import time
+import sys
+import platform
+import copy
+
+import xgboost
+# pylint: disable=no-name-in-module
+from xgboost import DMatrix
+
+from ..context import Context
+from ..log_capture import SystemOutputStreamMonitor
+from .. import create_run
+
+DEFAULT_EVALS_NAME = 'Test Set'
+DEFAULT_RUN_OPTIONS = {
+  'log_sys_info': True,
+  'log_stdout': True,
+  'log_stderr': True,
+  'run': None
```

Comment:
What's this `run` for again?

Reply:
Ok...I see what you have below...I'm not sure if this is necessary.

Reply:
oh hmm....I totally missed that in the API spec doc. Let me read it more carefully.